alacritty/src/event.rs

```diff
@@ -1201,6 +1202,13 @@ impl input::Processor<EventProxy, ActionContext<'_, Notifier, EventProxy>> {
                         self.ctx.update_cursor_blinking();
                         self.on_focus_change(is_focused);
                     },
+                    WindowEvent::Occluded(occluded) => {
+                        *self.ctx.occluded = occluded;
+                        if !occluded {
+                            // Render as soon as the window is visible once again.
+                            *self.ctx.dirty = true;
+                        }
```

Comment:
I'm pretty sure this not required. Since to reset `dirty` you must be not `occluded`. Is it causing issues when the window wasn't changed while being occluded and then present to the display?

Reply:
Yes sorry I realised it probably redundant last night and was going to remove it. For some reason my brain thought if it can now be seen then it needs to redraw but that obviously isn't the case. I'll remove it.